test/index.js

```diff
@@ -20,4 +20,30 @@ describe('analytics', function () {
       assert(a.name == b.name);
     });
   });
+
+  describe('noConflict', function () {
+    beforeEach(function () {
+      analytics = window.analytics;
+    });
+
+    afterEach(function () {
+      window.analytics = analytics;
+    });
+
+    it('should leave window.analytics unchanged', function () {
+      // Overwrite window.analytics with random object
+      var dummy = {'dummy': true};
+      window.analytics = dummy;
+      assert(window.analytics === dummy);
+
+      // Unfortunately loading scripts in doesn't work inside a test
+      window.analytics = require('../lib/index.js');
+      assert(window.analytics !== dummy);
+
+      var newAnalytics = window.analytics;
+      var noConflictAnalytics = window.analytics.noConflict();
+      assert(window.analytics === dummy);
```

Comment:
I added checks to confirm that dummy objects remains as is mostly so that the test flows and tells you exactly what it's expecting.

Do you still think I should remove that?